keras/src/saving/file_editor.py

```diff
@@ -549,6 +550,158 @@ def _weights_summary_cli(self):
 
     def _weights_summary_iteractive(self):
 
+        def _find_factors_closest_to_sqrt(num):
```

Comment:
Please move the utility function out of the class (e.g. to the bottom of the file)

Reply:
I changed these code's location to bottom.